### PR TITLE
Updating Laravel.gitignore

### PR DESCRIPTION
**Pull request by amiteshbhartiya**
**Reasons for making this change:**

Updating .gitignore for laravel to add vagrent, vscode, node related local files

### CODE REVIEW

File: Laravel.gitignore [hunk 1]
```diff
@@ -18,3 +18,13 @@ Homestead.json
 
 # Rocketeer PHP task runner and deployment package. https://github.com/rocketeers/rocketeer
 .rocketeer/
+yarn-error.log
+
+#The .idea folder (hidden on OS X) in the solution root contains IntelliJ’s project specific settings files
+/.idea
```
Comment (fabiofdsantos) [inline Laravel.gitignore]: Why including this line if it's not related with the Laravel Framework?

Comment (shiftkey) [reply Laravel.gitignore]: This is covered by [`Global\JetBrains.gitignore`](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore).

File: Laravel.gitignore [hunk 2]
```diff
@@ -18,3 +18,13 @@ Homestead.json
...
+
+#Visual Studio Code project specific setting
+/.vscode
```
Comment (shiftkey) [inline Laravel.gitignore]: This is covered by [`Global\VisualStudioCode.gitignore`](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore).

File: Laravel.gitignore [hunk 3]
```diff
@@ -18,3 +18,13 @@ Homestead.json
 
 # Rocketeer PHP task runner and deployment package. https://github.com/rocketeers/rocketeer
 .rocketeer/
+yarn-error.log
```
Comment (shiftkey) [inline Laravel.gitignore]: I just merged #2772 which added this rule

File: Laravel.gitignore [hunk 4]
```diff
@@ -18,3 +18,13 @@ Homestead.json
...
+
+#The .idea folder (hidden on OS X) in the solution root contains IntelliJ’s project specific settings files
+/.idea
+
+#Visual Studio Code project specific setting
+/.vscode
+
+#vagrant specific file
+/.vagrant
```
Comment (shiftkey) [inline Laravel.gitignore]: I just merged #2772 which added this rule

**Comment by shiftkey** [comment]
@amiteshbhartiya I'm going to close this out because I believe the changes are already covered by other templates, or were merged recently.

**Pull request closed without merging**